src/prompts.js

```diff
@@ -8,6 +8,13 @@ module.exports = function () {
       message: 'Does your app have TypeScript support?',
       default: false,
     },
+    {
+      name: 'gqlTag',
+      type: 'confirm',
+      message:
+        'Will you use `gql` template literal tag to define queries? See https://github.com/apollographql/graphql-tag',
+      default: true,
+    },
```

Comment:
I prefer this:
```ts
import { gql } from '@apollo/client/core'
```

and the Apollo team recommends it too: https://www.apollographql.com/docs/react/migrating/apollo-client-3-migration/#graphql-tag

So, my recommendation would be to not have this prompt at all. If they really need to explicitly install and import from `graphql-tag` for whatever reason, then they probably know some stuff and can do it themselves.

Reply:
That actually makes a lot of sense, since `@apollo/client/core` just re-export `graphql-tag`
Updating it now